### ISSUE TITLE
Play Video in Background (Android)

### ISSUE THREAD

**Issue by abhi-m-jith** [recloudstream/cloudstream#110]
### Play videos in background
A feature to enable playing the video in background while the device screen is turned off. It would be a great thing if such a feature could be implemented. I have tried to copy the stream link and played it in VLC app while using this feature.
![image](https://user-images.githubusercontent.com/83087087/190418887-6ce8cc8b-623a-4495-8006-66f1d41decd5.png)

This will allow playing documentaries or such shows to be enjoyed like podcasts.
I looked into the code but I'm not familiar with Kotlin , so such an addition would be greatly appreciated and I believe there are other users who would also like such an option. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by LagradOst** [comment]
No, this was tried, but due to some exoplayer bugs I was not able to get it working
